tests/providers/azure/services/storage/storage_key_rotation_90_days/storage_key_rotation_90_days_test.py

```diff
@@ -0,0 +1,144 @@
+from unittest import mock
+from uuid import uuid4
+
+from prowler.providers.azure.services.storage.storage_service import Storage_Account
+from tests.providers.azure.azure_fixtures import AZURE_SUSCRIPTION
+
+
+class Test_storage_key_rotation_90_dayss:
+    def test_storage_no_storage_accounts(self):
+        storage_client = mock.MagicMock
+        storage_client.storage_accounts = {}
+
+        with mock.patch(
+            "prowler.providers.azure.services.storage.storage_key_rotation_90_days.storage_key_rotation_90_days.storage_client",
+            new=storage_client,
+        ):
+            from prowler.providers.azure.services.storage.storage_key_rotation_90_days.storage_key_rotation_90_days import (
+                storage_key_rotation_90_days,
+            )
+
+            check = storage_key_rotation_90_days()
+            result = check.execute()
+            assert len(result) == 0
+
+    def test_storage_storage_key_rotation_91_days(self):
```

Comment:
Why more than 90 days is a FAIL? Just to understand.

Reply:
I took the recomendation from https://www.trendmicro.com/cloudoneconformity-staging/knowledge-base/azure/StorageAccounts/regenerate-storage-account-access-keys-periodically.html#